src/server/components/lists/helpers.ts

```diff
@@ -1,6 +1,6 @@
 import querystring from "querystring";
 import { Express, Request } from "express";
-import { get, omit, trim, mapKeys, isArray, without, lowerCase, kebabCase, camelCase } from "lodash";
+import _, { get, omit, trim, mapKeys, isArray, without, lowerCase, kebabCase, camelCase } from "lodash";
```

Comment:
Was the underscore added to make it obvious that lodash was being used?

Reply:
fair point - just a habit using _.  I've fixed this one.